### ISSUE TITLE
Unstackable field group don't have any margin on mobile

### ISSUE THREAD

**Issue by gaku-sei** [fomantic/Fomantic-UI#1131]
# Bug Report

Not sure if it's a bug, or if I'm just misusing the library. The margin bottom for unstackable form group is set to `0` on mobile which leads to inconsistent UI.

Notice that the margin appears on desktop.

## Code

```html
<div class="unstackable fields">
  <div class="ten wide field">
    <div class="field">
      <label>業界</label>
      <div role="listbox" aria-expanded="false" class="ui selection dropdown" tabindex="0">
        <div class="default text" role="alert" aria-live="polite" aria-atomic="true">選択してください</div>
        <i aria-hidden="true" class="dropdown icon"></i>
        <div class="menu transition"></div>
      </div>
    </div>
  </div>
  <div class="six wide field">
    <div class="field">
      <label>今年の採用目標数</label>
      <div class="ui input"><input type="text"></div>
    </div>
  </div>
</div>
```

## Expected result

Homogeneous margin.

<img width="326" alt="Capture d’écran 2019-10-29 à 17 00 11" src="https://user-images.githubusercontent.com/3982371/67748160-9afa8780-fa6d-11e9-9171-c210aa09bea5.png">

## Actual result

No margin bottom.

<img width="327" alt="Capture d’écran 2019-10-29 à 16 59 21" src="https://user-images.githubusercontent.com/3982371/67748092-7a323200-fa6d-11e9-8eb3-5aab229f50f2.png">

## Version
fomantic-ui-css 2.7.8

**Comment by hammy2899** [comment]
@gaku-sei Could you please create a [JSFiddle](https://jsfiddle.net/31d6y7mn) to help us replicate your issue.

**Comment by lubber-de** [comment]
I think you mixed up some components here 😉 
- It should be `ui unstackable grid` instead of `unstackable fields`
- It should be `wide column` instead of `wide field`

I prepared a little jsfiddle. If those hints do not help you to solve your issue, please adjust the example, save it in jsfiddle again and provide a reproducable testcase. Thank you 🙂 

https://jsfiddle.net/9f7ovmdc/

**Comment by gaku-sei** [comment]
@lubber-de Thank you very much for your answer! I used the grid component at first, but wondered if the field related components could help achieving the same thing.
So for this kind of use cases, grid seems the way to go then 👍

**Comment by lubber-de** [comment]
I think i mislead your issue back then. Using field groups had indeed wrong bottom margin, which should be fixed by #1550 now
